components/Snackbar/src/MDCSnackbarMessageView.h

```diff
@@ -71,7 +71,7 @@
 /**
  Returns the button title color for a particular control state.
 
- Default for UIControlStateNormal is MDCRGBAColor(0xFF, 0xFF, 0xFF, 0.6f).
+ Default for UIControlStateNormal is MDCRGBAColor(0xFF, 0xFF, 0xFF, (CGFloat)0.6).
```

Comment:
tip: remove casting, make it just 0.6

Reply:
Same as above, I wanted to preserve the syntactic correctness of the code.